src/components/Share/style.scss

```diff
@@ -0,0 +1,30 @@
+.share {
```

Comment:
+0
It's great that U did not use any useless css attribute/property but we can improve code by using DRY principle and module specific css selector(that support future extension)
```scss
.surah-share{
  &-container{
     position: relative;
     bottom: 2px;
     left: 5px;
  }
 &-icon{
   background-repeat: no-repeat;
   padding-right: 30px;
   background-size: 12px;
   padding-bottom: 10px;
   padding-top: 1px;
 }
 &-facebook{
    @extend .surah-share-icon;
    background-image: url(../../../static/images/FB-grn.png);
    &:hover{
      background-image: url(../../../static/images/FB-beige.png); 
    }
  }
   &-twitter{
    @extend .surah-share-icon;
    background-image: url(../../../static/images/Twitter-grn.png);
    &:hover{
      background-image:  url(../../../static/images/Twitter-beige.png); 
    }
  }
} 
```
Some benefits of above approach:
- code readability
- css size is reduced
- if implement share functionality at different level(e.g. Ayah level), we can define a different select(e.g. ayah-share-facebook) without conflicting with a generic ```.facebook``` selector.